src/common/storage/src/operator.rs

```diff
@@ -104,14 +105,16 @@ pub fn build_operator<B: Builder>(builder: B) -> Result<Operator> {
         .layer(
             TimeoutLayer::new()
                 // Return timeout error if the operation failed to finish in
-                // 60s
-                .with_timeout(Duration::from_secs(60))
+                // 10s
+                .with_timeout(Duration::from_secs(10))
                 // Return timeout error if the request speed is less than
                 // 1 KiB/s.
                 .with_speed(1024),
         )
         // Add retry
         .layer(RetryLayer::new().with_jitter())
+        // Add async backtrace
+        .layer(AsyncBacktraceLayer)
```

Comment:
Can we configure disable?

Reply:
I'm not sure. `async-backtrace` in opendal deesn't collect or print, we just add bunch of `async_backtrace::framed`. So it's controlled by databend instead.

Reply:
Merge now or not?

Reply:
Let'e merge.